install_config/storage_examples/privileged_pod_storage.adoc

```diff
@@ -0,0 +1,206 @@
+link:../admin_guide/manage_authorization_policy.html#managing-role-bindings[*cluster-admin*] role binding.
+** For this guide, that user is called `admin`
+
+
+== Make the Volume Accessible
+
+[[create-the-pv]]
+
+=== Create the persistent volume
+
+Creating the *PersistentVolume* will make the storage accessible to users,
+regardless of namespace.
+
+.*As admin*
+[source]
+----
+$ oc create -f gluster-endpoints-service.yaml
+$ oc create -f gluster-endpoints.yaml
+$ oc create -f gluster-pv.yaml
+----
+
+Verify Success
+
+[source]
+----
+$ oc get pv
+NAME                     LABELS    CAPACITY   ACCESSMODES   STATUS      CLAIM     REASON    AGE
+gluster-default-volume   <none>    2Gi        RWX           Available                       2d
+----
+----
+$ oc get svc
+NAME              CLUSTER_IP      EXTERNAL_IP   PORT(S)   SELECTOR   AGE
+gluster-cluster   172.30.151.58   <none>        1/TCP     <none>     24s
+----
+----
+$ oc get ep
+NAME              ENDPOINTS                           AGE
+gluster-cluster   192.168.59.102:1,192.168.59.103:1   2m
+----
+
+[[create-regular-user]]
+== Create a regular user
+
+A link:../../architecture/core_concepts/projects_and_users#users[*regular user*] must be added to the privileged scc (or to a group given access
+  to the scc) before they can run privileged pods.
+
+.*As admin*
+[source]
+----
+oadmin policy add-scc-to-user privileged tom
+----
+
+Then login to the regular user.
+
+[source]
+----
+$ oc login -u tom -p tom
+----
+
+NOTE: Here, `tom` is an arbitrary username and password
+
+Next, create the project.
+
+[source]
+----
+$ oc new-project <project_name>
+----
+
+[[create-the-pvc]]
+=== Create the persistent volume claim
+
+.*As regular user*
+
+Create the *PersistentVolumeClaim* to access the volume.
+
+[source]
+----
+$ oc create -f gluster-pvc.yaml -n <project name>
+----
+
+[[create-pod]]
+=== Define the Pod
+
+First, define your pod to access the claim.
+
+.Pod Definition
+====
+
+[source,yaml]
+----
+apiVersion: v1
+id: gluster-nginx-pvc
+kind: Pod
+metadata:
+  name: gluster-nginx-priv
+spec:
+  containers:
+    - name: gluster-nginx-priv
+      image: fedora/nginx
+      volumeMounts:
+        - mountPath: /mnt/gluster <1>
+          name: gluster-volume-claim
+      securityContext:
+        privileged: true
+  volumes:
+    - name: gluster-volume-claim
+      persistentVolumeClaim:
+        claimName: gluster-claim <2>
+
+----
+<1> Volume mount within the pod
+<2> Must reflect the name of the *PersistentVolume*
+====
+
+=== Create the pod
+
+On pod creation, the mount directory will be created and the volume attached
+to that mount point.
+
+.*As regular user*
+
+[source]
+----
+$ oc create -f gluster-nginx-pod.yaml
+----
+
+Check to make sure the pod created successfully.  It can take several minutes for the pod to create.
+
+[source]
+----
+$ oc get pods
+NAME                 READY     STATUS    RESTARTS   AGE
+gluster-nginx-pod   1/1       Running   0          36m
+----
+
+[[verify]]
```

Comment:
IDs should aim for uniqueness across the set of books (i.e., the full openshift-docs repo).